### DISCUSSION TITLE
Compilation Error-Resolved

### DISCUSSION THREAD

**Discussion by jbatesmd** [cyberman54/ESP32-Paxcounter#808] [General]
It fails to compile with this error src\blecsan.cpp:15:22: fatal error: bt_types.h: No such file or directory.  I assume I'm missing a bluetooth library but not sure any ideas?  I'm compiling for the TTGO T3 v1.6

**Comment by cyberman54** [comment]
I can't reproduce this, and the file bt_types.h is no longer used by paxcounter software.
Did you
- use the current version 3.0.0 in master branch?
- press "Clean" button in PlatformIO beforde build?

**Comment by jbatesmd** [comment]
I found the 3.0 version and got it to compile thank you for responding to me.  It’s working fine now.

Johnny E Bates MD


On May 31, 2021, at 11:39 AM, Verkehrsrot ***@***.***> wrote:

﻿

[ WARNING! ]
This email originated from outside the QCHC email system. DO NOT click links or open attachments unless you recognize the sender and know the content is safe.

I can't reproduce this, and the file bt_types.h is no longer used by paxcounter software.
Did you

  *   use the current version 3.0.0 in master branch?
  *   press "Clean" button in PlatformIO beforde build?

**Reply by cyberman54** [reply]
Great! Would please mark this thread as solved? Thx.
